src/server.ts

```diff
@@ -155,6 +157,9 @@ app.use((req, res, next) => {
     next();
 });
 
+
+app.use(basePath + '/lti', lti.app);
```

Comment:
this is dangerous
I'm not sure but I feel like if base path ends in `/` this might break